endpoints.tf

```diff
@@ -68,3 +68,18 @@ data "template_file" "local_metastores_yaml" {
     mapped_databases = "${ lookup(var.local_metastores[count.index],"mapped-databases","") }"
   }
 }
+
+resource "aws_route53_zone" "remote_metastore" {
+  count = "${ var.enable_remote_metastore_dns == "" ? 0 : 1 }"
+  name = "${local.remote_metastore_zone_prefix}-${var.region}.lcl"
+  vpc_id = "${var.vpc_id}"
+}
+
+resource "aws_route53_record" "hms_readwrite_alias" {
```

Comment:
can we have an alias for `hms_readonly` as well?

Reply:
this is copy and paste error, variable should be just named metastore_alias.
this is alias of remote metastore, it can be readonly or read/write.